kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Utils.java

```diff
@@ -130,7 +130,9 @@ public static boolean waitUntilReady(BlockingQueue<Object> queue, long amount, T
       if (obj instanceof Boolean) {
         return (Boolean) obj;
       } else if (obj instanceof Throwable) {
-        throw (Throwable) obj;
+        Throwable t = (Throwable) obj;
+        t.addSuppressed(new Throwable("waiting here"));
```

Comment:
Could we be more expressive in the message inside throwable? Or does this message has not any significance?